core/configuration.py

```diff
@@ -248,6 +259,20 @@ def process_section(section, subsections=None):
             elif section in CFG_OLD.keys():
                 process_section(section, subsection)
 
+        # migrate SiCRKAGE settings from SickBeard section to new dedicated SiCRKAGE section
+        if CFG_OLD['SickBeard']['tv']['enabled'] and CFG_OLD['SickBeard']['tv']['fork'] == 'sickrage-api':
+            for option, value in iteritems(CFG_OLD['SickBeard']['tv']):
+                if option in CFG_OLD['SiCKRAGE']['tv']:
```

Comment:
@echel0n 
Should this line be `if option in CFG_NEW['SiCKRAGE']['tv']:`
i.e. we only copy in the new item if it exists in the NEW cfg template. Since we are parsing the items in the CFG_OLD, we know they the items does exist already...